README.md

```diff
@@ -47,10 +47,10 @@ export default Ember.Controller.extend({
 
 The `cookies` service has methods for reading and writing cookies:
 
-* `read(name)`: reads the cookie with the given name, returns its value as a
-  `String`.
+* `read(name, options = {})`: reads the cookie with the given name, returns its value as a
+  `String`; options can be used to set `useRawValue` (boolean).
 * `write(name, value, options = {})`: writes a cookie with the given name and
-  value; options can be used to set `domain`, `expires` (Date), `maxAge` (time in seconds), `path` and `secure`.
+  value; options can be used to set `domain`, `expires` (Date), `maxAge` (time in seconds), `path`, `secure`, and `useRawValue` (boolean).
```

Comment:
we should add something like _"…and `raw` which disables (boolean) URL-encoding the value"_